.github/workflows/check.yaml

```
@@ -0,0 +1,19 @@
+name: Check
+on: [push]
+jobs:
+  Check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 17
+          cache: npm
+      - name: Dependencies
+        run: npm ci
+      - name: Lint
+        run: npm run lint
+      - name: Test
+        run: npx jest
```

As you use `yarn` in your project, for consistency, use it in `CI` as well

```suggestion
      - name: Dependencies
        run: yarn install --immutable --immutable-cache --check-cache
      - name: Lint
        run: yarn lint
      - name: Test
        run: yarn test
```